README.md

```diff
@@ -1,14 +1,14 @@
 # Frappe on Docker
 
-[![Build Status](https://travis-ci.org/frappe/frappe_docker.svg?branch=develop)](https://travis-ci.org/frappe/frappe_docker)
+[![Build Status](https://travis-ci.com/chabad360/frappe_docker.svg?branch=develop)](https://travis-ci.com/chabad360/frappe_docker)
```

Comment:
I think you should reset this entire file.

Reply:
As in rewrite the whole thing?

Reply:
I'll definitely fix that (oopsie), but a rewrite (at least now) seems a bit overboard. The README is pretty good for what the project is now, but once I get merged the rest of my changes, then we'll have what to talk about.

Reply:
no i meant you should reset it back to original.